frontend/src/components/FeaturedStories.jsx

```diff
@@ -2,111 +2,44 @@ import React, { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import axios from 'axios';
 
+// Helper function to truncate text to a maximum of 50 words
```

Comment:
I would recommend having all helper functions related to text cleaning, processing in a single file